playground/nextjs/src/pages/index.tsx

```diff
@@ -15,9 +15,6 @@ export default function Home() {
             '6697c185607852e9104daf33': 'My workflow in other language', // key is workflow id
           },
         }}
-        appearance={{
-          baseTheme: dark,
-        }}
```

Comment:
This is demonstrated in the Themes page of the playground app. It's useful to have light theming on the main page for recording demonstrations.